test/bdd/cmd/issuer/go.mod

```diff
@@ -0,0 +1,12 @@
+// Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
can we use httptest.Server instead of adding more docker images to the build?

Reply:
I just added `go.mod` - not a new docker

Reply:
right - my bad